src/terrain/generation/biome.hpp

```diff
@@ -135,7 +135,20 @@ class Biome {
      *
      * @return 2D map of map tiles
      */
-    [[nodiscard]] const std::vector<MapTile> get_map(MacroDim length) const;
+    [[nodiscard]] const TerrainMacroMap get_map(MacroDim length) const;
+
+    inline static TerrainMacroMap
+    get_test_map(MapTile_t type) {
```

Comment:
no test code

Reply:
What do you mean by this? This code is used to test json files. This is needed in production as it will be used to test mods.

Reply:
ok then give it a more descriptive name